.travis.yml

```diff
@@ -131,6 +131,14 @@ script:
         echo -e "Variable HOST_CLOUD not set\n"
     fi
 
+  # Check whether this build was triggered by our cron job for security-updates
+  - echo "TRAVIS_EVENT_TYPE is:" && echo $TRAVIS_EVENT_TYPE
```

Comment:
`echo "TRAVIS_EVENT_TYPE is: $TRAVIS_EVENT_TYPE` is more readable IMO

Reply:
Sure, no problem.